docs/pages/eas-update/environment-variables.md

```diff
@@ -2,36 +2,32 @@
 title: Using environment variables with EAS Update
 ---
 
-> ⚠️ The APIs below are not stable and are expected to change as we develop EAS Update. An alternate solution is to use a babel plugin to transform variables into environment variables, as described [here](/guides/environment-variables/#using-babel-to-replace-variables).
-
-EAS Build and EAS Update allow setting and getting environment variables at different times. There are multiple steps to ensure the proper environment variables are available when developing, building, and publishing an update to a project.
+EAS Build and EAS Update allow setting and getting environment variables at different times. There are multiple steps to ensure the proper environment variables are available when developing, building, and publishing an update.
 
 ## Setting up the app config
 
 To expose environment variables in our project, we'll have to set up our app config to allow it. After this, we'll be able to access these variables throughout our entire project.
 
 First, we'll need to rename **app.json** to **app.config.js**, which will allow us to export our app's config, including JavaScript variables.
 
-Next, we'll add an `API_URL` environment variable to our project.
+Next, we'll add an `API_URL` environment variable (for example) to our project.
```

Comment:
```suggestion
Next, we'll add an `API_URL` environment variable as an example to our project.
```

"We" as a team have to stick to using either `you` or `we`. I don't think its helping us to keep docs consistent. 😄 I also know this is not related to your PR but was wondering what thoughts you have.